documentation/modules/ref-ability-restrict-address-space-connections.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring.adoc
+
+[id='ref-ability-restrict-address-space-connections-{context}']
+= Ability to restrict address space connections
+
+Company X has the following requirement for using {ProductName}: For engineering work, the ability to restrict who can connect into the address space is required.
+
+To meet this requirement you must set a network policy in the brokered infrastructure configuration. For more information about network policies, see OpenShift documentation.
```

Comment:
Is it possible to link to Kubernetes/OpenShift docs here?

Reply:
My bad; will push a fix